frontend/public/actions/features.ts

```diff
@@ -174,14 +174,24 @@ export type FeatureAction = Action<
 >;
 
 const openshiftPath = '/apis/apps.openshift.io/v1';
-const detectOpenShift = (dispatch) =>
+const detectOpenShift = (dispatch) => {
   fetchURL(openshiftPath).then(
-    (res) => dispatch(setFlag(FLAGS.OPENSHIFT, _.size(res.resources) > 0)),
-    (err) =>
-      err?.response?.status === 404
-        ? dispatch(setFlag(FLAGS.OPENSHIFT, false))
-        : handleError(err, FLAGS.OPENSHIFT, dispatch, detectOpenShift),
+    (res) => {
+      const isOpenShift = _.size(res.resources) > 0;
+      const hasMonitoring = isOpenShift && !!window.SERVER_FLAGS.prometheusBaseURL;
+      dispatch(setFlag(FLAGS.OPENSHIFT, isOpenShift));
+      dispatch(setFlag(FLAGS.MONITORING, isOpenShift && hasMonitoring));
```

Comment:
@spadgett @christianvogt Is it okay to calculate a second flag here? It can then be used to filter nav items automatically? The alternative would be a dynamic `FeatureFlag`, but I thought it was a little bit to much for a `SERVER_FLAGS` check, wdyt?

Reply:
would it make sense to update the `FLAGS` constant and set this flag in the `defaults` section of `features.ts` similar to how `AUTH_ENABLED` is initialized?

Reply:
Sure, this makes sense, doesn't check the default values in the reducer. Thanks for that hint.

And yes, it depends already on both flags. I will test and update the PR.